styles/signals.mapcss

```diff
@@ -730,13 +737,11 @@ node|z16-[railway=signal]["railway:signal:direction"]["railway:signal:train_prot
 	icon-height: 20;
 	allow-overlap: true;
 }
-node|z16-17[railway=signal]["railway:signal:direction"]["railway:signal:train_protection"="DE-ESO:blockkennzeichen"]::blockkennzeichen
+node|z16-[railway=signal]["railway:signal:direction"]["railway:signal:train_protection"="DE-ESO:blockkennzeichen"]::blockkennzeichen
 {
 	z-index: 8510;
 	text: "ref";
 	text-offset-x: -28;
-	kothicjs-ignore-text-offset-x: true;
-	text-offset-y: 12;
 	text-color: black;
 	text-halo-radius: 1;
 	text-halo-color: white;
```

Comment:
You could remove `text-halo-color` and `text-halo-radius` as the background of the caption is a white sign.